idd/Energy+.idd.in

```diff
@@ -87021,7 +87064,7 @@ UtilityCost:Qualify,
         \memo "Qualified" entry will say "No" and the UtilityCost:Qualify that caused its exclusion
         \memo is shown. Multiple UtilityCost:Qualify objects can appear for the same tariff and
         \memo they can be based on any variable.
-   A1,  \field Name
+   A1,  \field Utility Cost Qualify Name
```

Comment:
So objects where the Name fields were renamed was due to how our automated translation from IDF to JDF works. If there is a Name field, it becomes a key and thus must be unique by the JSON standard. What happens is that in some cases, objects have a Name field but the name can be non-unique. Thus the easiest and most straightforward way to address this was to change the Name field to a more specific field name.